### PR TITLE
Add HTML and CSS files, update Readme.md

### PR DESCRIPTION
**Pull request by LYANGEND**
**Done the following**:
-Add HTML and CSS file
-Update README files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse survey form</title>
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+    <div>
+    <h1 id=title>Microverse Survey form</h1>
```
Comment (LYANGEND) [inline index.html:11]: Nice job designing the survey form but the heading is too close to the main content, kindly adjust it accordingly.

File: README.md [hunk 2]
```diff
@@ -24,33 +26,14 @@
...
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
```
Comment (LYANGEND) [inline README.md:39]: Your readme file has too many empty sections, I suggest you could either fill in those sections with information about your project or simply remove them in order to make your Readme file more professional.

File: README.md [hunk 3]
```diff
@@ -24,33 +26,14 @@
...
 
 ## Authors
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
```
Comment (LYANGEND) [inline README.md:48]: Please fill in the details of the author.

**Review by LYANGEND — COMMENTED** [review]
# Changes required:

Please check my comments below and make the required changes
Submit for a re-review when you are done.
Thank yoou.

**Review by gemmen29 — APPROVED** [review]
Hi @LYANGEND, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**